patches/minecraft/net/minecraft/client/renderer/entity/EntityRenderer.java.patch

```diff
@@ -0,0 +1,23 @@
+--- a/net/minecraft/client/renderer/entity/EntityRenderer.java
++++ b/net/minecraft/client/renderer/entity/EntityRenderer.java
+@@ -15,6 +15,7 @@
+ 
+ @OnlyIn(Dist.CLIENT)
+ public abstract class EntityRenderer<T extends Entity> {
+@@ -54,9 +55,14 @@
+    }
+ 
+    public void func_225623_a_(T p_225623_1_, float p_225623_2_, float p_225623_3_, MatrixStack p_225623_4_, IRenderTypeBuffer p_225623_5_, int p_225623_6_) {
+-      if (this.func_177070_b(p_225623_1_)) {
+-         this.func_225629_a_(p_225623_1_, p_225623_1_.func_145748_c_().func_150254_d(), p_225623_4_, p_225623_5_, p_225623_6_);
++
++      net.minecraftforge.client.event.RenderNameplateEvent renderNameplateEvent = new net.minecraftforge.client.event.RenderNameplateEvent(p_225623_1_,p_225623_1_.func_145748_c_().func_150254_d());
++      net.minecraftforge.common.MinecraftForge.EVENT_BUS.post(renderNameplateEvent);
++
++      if (!renderNameplateEvent.getResult().equals(Event.Result.DENY) && (renderNameplateEvent.getResult().equals(Event.Result.ALLOW) || this.func_177070_b(p_225623_1_))) {
```

Comment:
Just use `==` for enums.

Reply:
Also, this doesn't even compile, because `Event` is not a thing here... How on earth did you generate this patch?

Reply:
Replaced equals with == operator now